python/aswfdocker/data/versions.yaml

```diff
@@ -828,6 +831,7 @@ groups:
         - gtest
         - blosc
       base2:
+        - expat
```

Comment:
Not a blocker, but I would consider `expat` a more generic dependency than just for qt, for instance OCIO looks for it in its `Config.cmake.in`, so I'd move expat to the `base1` build stage.

AFAIK the reason why `qt` and `pyside` are in separate `base2` and `base3` sections is due to the build time which surpases what can be done on a base GHA runner. `CONTRIBUTING.md` documents how to upload a release created from a local build, but now that we have access to larger runners, it is preferable to build everything using GHA.

Reply:
I agree, I can move it to base1.